website/conferences/views.py

```diff
@@ -77,6 +77,9 @@ def add_poster_by_email(conference, message):
             is_spam=message.is_spam,
         )
         if user_created:
+            if utils.is_valid_email(user.fullname):
+                user.fullname = user._id  # Users cannot use an email as their full name
```

Comment:
This works, but it's a bit awkward. Why not change the `fullname` argument passed to `get_or_create_user`?

Reply:
We don't have the guid prior to user creation. We could do something different, but using the guid is a Product choice.

Reply:
Ah ok. This is fine, then.